corehq/apps/accounting/tests/generator.py

```diff
@@ -39,9 +38,11 @@
     SmsUsageFeeCriteria,
 )
 from corehq.apps.users.models import WebUser, CommCareUser
+from corehq.util.test_utils import unit_testing_only
```

Comment:
seems kind of odd to use this decorator now that the module is inside a `tests` package.

I think this decorator maybe shouldn't exist since the issue it was designed to protect against is an anti-pattern (mixing test code with production code). I guess I can see a case for it during a transitional period of moving test code out of production modules. Test code should always reside inside a test package and should never be imported by production code.